server/src/test/java/org/elasticsearch/indexlifecycle/IndexLifecycleActionIT.java

```diff
@@ -67,7 +67,8 @@ public void testIndexLifecycleActionsWith11Shards1Backup() throws Exception {
         final String node1 = getLocalNodeId(server_1);
 
         logger.info("Creating index [test]");
-        CreateIndexResponse createIndexResponse = client().admin().indices().create(createIndexRequest("test").settings(settings)).actionGet();
+        CreateIndexResponse createIndexResponse = client().admin().indices().create(
```

Comment:
I noticed that you break lines on '(' . Was that an intention? Reminds me of my early projects in Lisp :) I am worried that most of other refactoring would be breaking on chained method '.'
what do you think of breaking on '.'
for instance
```
client().admin().indices()
.create(reateIndexRequest("test").settings(settings)))
.actionGet();
```
or even one '.' per each line - although that might have been excessive

Reply:
The intention was to separate the request parameter from the actual call.